tests/test_vector.py

```diff
@@ -71,9 +80,15 @@ def test_vector_metadata(self):
             self.assertEqual(v.meta, expected)
 
         with self.subTest("codec meta"):
-            simd = SIMD(self.source, self.output1)
-            x = v > simd
-            self.assertEqual(x.meta, expected)
+            a = self.simd.audio
```

Comment:
This is a smoke test for `Vector.meta` property. 
* look at `FilterGraphTestCase`
* there is no need to add extra output to `SIMD` here, just add bitrate parameter in `setUp`

Reply:
Updated `vector_metadata` test and added tests to `FilterGraphTestCase` including some cases to fix coverage.